src/ECharts/Style/Area.purs

```diff
@@ -0,0 +1,32 @@
+module ECharts.Style.Area where
+
+import Data.Maybe
+import Data.StrMap (fromList, StrMap (..))
+import Data.Tuple
+import Data.Argonaut.Core
+import Data.Argonaut.Encode
+import Data.Argonaut.Combinators
+
+import ECharts.Color
+
+data AreaType = Fill
```

Comment:
If this is the only option, why does this even exist? Or are there other options besides "fill"?